apps/main/src/loan/components/PageLoanManage/LoanManageSoftLiq.tsx

```diff
@@ -0,0 +1,138 @@
+import { useCallback, useMemo, useState } from 'react'
+import { useChainId } from 'wagmi'
+import {
+  ManageSoftLiquidation,
+  type ActionInfosProps,
+  type ClosePositionProps,
+  type ImproveHealthProps,
+} from '@/llamalend/features/manage-soft-liquidation'
+import {
+  checkCanClose,
+  getCollateralInfo,
+  getCollateralToRecover,
+  getDebtToken,
+  getHealthInfo,
+  getLoanInfo,
+  getTokens,
+  parseFloatOptional,
+} from '@/llamalend/features/manage-soft-liquidation/helpers'
+import { useClosePositionMint } from '@/llamalend/features/manage-soft-liquidation/hooks/mutations/useClosePosition'
+import { useRepayMint } from '@/llamalend/features/manage-soft-liquidation/hooks/mutations/useRepay'
+import { useUserLoanDetails } from '@/loan/hooks/useUserLoanDetails'
+import useStore from '@/loan/store/useStore'
+import type { MintMarketTemplate } from '@curvefi/llamalend-api/lib/mintMarkets'
+import { useTokenUsdRate } from '@ui-kit/lib/model/entities/token-usd-rate'
+
+type Props = {
+  market: MintMarketTemplate | undefined
+}
+
+/** Main component hooking up the manage soft liquidation card. Preferably only this parent component uses loan app specific imports. */
+export const LoanManageSoftLiq = ({ market }: Props) => {
```

Comment:
We don't need to abbreviate :nerd_face: 
```suggestion
export const LoanManageSoftLiquidation = ({ market }: LoanManageSoftLiquidationProps) => {
```